.editorconfig

```diff
@@ -9,3 +9,7 @@ insert_final_newline = true
 
 [*.md]
 trim_trailing_whitespace = false
+
+[*.test.ts]
+trim_trailing_whitespace = false
```

Comment:
It took me forever to figure out why my IDE kept removing spaces from test table expected outputs.